### DISCUSSION TITLE
社会科学数据资源汇总（CGSS、CSS、CLDS、CHIP、CFPS、CHARLS、CHFS、CLHLS、CHNS）

### DISCUSSION THREAD

**Discussion by chengjun** [data-journalism/data-journalism.github.io#64] [General]
本文由计量经济学服务中心综合整理

目前中国社区、家庭、个人层面的微观数据库，分别有北大中国社会科学调查中心，中山大学社会科学调查中心、中国人民大学中国调查与数据中心、清华大学中国经济社会数据中心、上海大学上海科学调查中心、西南财经大学中国家庭金融调查与研究中心、复旦大学社会科学数据研究中心、中国社科院调查与数据信息中心等。

下面就一起来看看微观经济数据库。

目录：

1、中国综合社会调查（CGSS）

2、中国社会状况综合调查（CSS）

3、中国劳动力动态调查（CLDS）

4、中国家庭收入项目（CHIP）

5、中国家庭追踪调查（CFPS）

6、中国健康与养老追踪调查（CHARLS）

7、中国家庭金融调查数据CHFS

8、全国老年人口健康状况调查研究（CLHLS）

9、中国健康与营养调查（CHNS）

其他数据库欢迎大家补充

调查中心（排名不分先后）

北京大学中国社会科学调查中心（Institute of Social Science Survey, ISSS）成立于2006年9月，是北京大学社会科学的数据调查平台，也是北京大学开展中国社会问题实证研究的跨学科平台。中心目前承担两个大型社会调查项目——中国家庭动态跟踪调查和中国健康养老追踪调查。两个项目的目的均是收集反映我国民生状况的高质量微观数据，用以分析社会民生方面的问题，为政策制定提供依据，同时推动社会、经济、教育等跨学科研究工作。

中国人民大学-中国调查与数据中心（National Survey Research Center at Renmin University of China, NSRC)是中国人民大学直属的跨学科、跨院系的综合性研究机构。中心的宗旨为科学、系统、全面地采集、整理、存储与开发中国经济与社会调查数据，进行调查方法与相关技术的研究开发，实施具有重大科学与现实意义的大型科研项目，为科学研究和政府决策提供数据支持。中国调查与数据中心（NSRC）围绕着中国的经济和社会数据，以数据采集、数据存储、数据开发为主要方向。中国调查与数据中心（NSRC）联合全国53所高校组成了“中国社会调查网络（CSSN）”，组织实施中国综合社会调查（CGSS）、中国教育追踪调查（CEPS）、中国宗教调查（CRS）、中国老年社会追踪调查(CLASS)、大学生成长追踪调查(CSDS)、千人百村调查等大型长期追踪调查项目，建成了大规模、全国性、全方位、多层次、连续性与自主性的社会科学基础数据采集平台。中国调查与数据中心（NSRC）受国家自然科学基金委托，建成了中国第一个社会科学调查数据资料库——中国国家调查数据库(CNSDA)，开创了我国社会科学数据开放与共享的先河。

中国社科院-社会学研究所、清华大学、中山大学、复旦大学、上海大学等也有各自的调查中心

# 1 中国综合社会调查数据CGSS

中国综合社会调查数据CGSS
中国综合社会调查（Chinese General Social Survey，CGSS）始于2003年，是我国最早的全国性、综合性、连续性学术调查项目。CGSS系统、全面地收集社会、社区、家庭、个人多个层次的数据，总结社会变迁的趋势，探讨具有重大科学和现实意义的议题，推动国内科学研究的开放与共享，为国际比较研究提供数据资料，充当多学科的经济与社会数据采集平台。目前，CGSS数据已成为研究中国社会最主要的数据来源，广泛地应用于科研、教学、政府决策之中。

2003-2008年是CGSS项目的第一期，共完成5次年度调查（2007年没有执行），生产出5套高质量的年度数据。除2004年的调查数据，剩下的年度数据都已在中国国家调查数据库（China National Survey Data Archive，CNSDA）的网站（cnsda.ruc.edu.cn）上发布，到目前为止，用户可免费申请使用。

执行机构丨中国人民大学中国调查与数据中心

数据网址丨http://www.cnsda.org/index.php?r=site/datarecommendation

开放数据年份丨2003、2005、2006、2008、2010、2011、2012、2013

时间跨度：分两期，第一期：2003年—2008年2008年，每年一次；第二期：2010年—2019年，每两年一次。最新公开数据：CGSS2013。

数据类型丨截面数据

分析单位丨个人、家庭

覆盖区域丨中国28个省市

核心问题丨中国社会变迁（文化、健康、家庭、劳动力、就业、消费、教育、心理、个性等）

应用主题丨人口健康分析、劳动就业分析、消费储蓄分析、空间规划分析\社会流动、幸福感、社会信任、教育回报、宗教信仰、政治参与等。

# 2 中国社会状况综合调查（CSS）
“中国社会状况综合调查”（Chinese Social Survey，简称CSS）是中国社会科学院社会学研究所于2005年发起的一项全国范围内的大型连续性抽样调查项目，目的是通过对全国公众的劳动就业、家庭及社会生活、社会态度等方面的长期纵贯调查，来获取转型时期中国社会变迁的数据资料，从而为社会科学研究和政府决策提供翔实而科学的基础信息。

该调查是双年度的纵贯调查，采用概率抽样的入户访问方式，调查区域覆盖了全国31个省/自治区/直辖市，包括了151个区市县，604个村/居委会，每次调查访问7000到10000余个家庭。此调查有助于获取转型时期中国社会变迁的数据资料，其研究结果可推论全国年满18－69周岁的住户人口。

为了兼顾纵贯调查的连续性和社会议题的现实性， CSS的调查问卷在设计上分为基础模块、更替模块和热点模块三个部分。其中基础模块固定不变，包含了个人基础信息、劳动与就业、家庭结构、家庭经济状况等内容；更替模块如社会阶层地位流动、社会保障、休闲消费、社会价值观等，隔一定周期后重复调查；热点模块则与时俱进，目前已进行了社会群体利益关系、民生问题、城镇化等主题的研究。

数据下载方式：

http://css.cssn.cn/css_sy/

# 3 中国劳动力动态调查数据CLDS
中国劳动力动态调查数据CLDS

执行机构丨中山大学社会科学调查中心

数据下载：可下载spss、stata格式的数据，下载的数据格式由数据原始格式决定。http://css.sysu.edu.cn/

数据网址丨http://css.sysu.edu.cn/Data

http://cus.sysu.edu.cn/sjku.asp?id=887

开放数据年份丨2011、2012、2014

数据类型丨面板数据

分析单位与调查规模丨社区、家庭、劳动；调查对象为样本家庭户中的全部劳动力（年龄15至64岁的家庭成员）。

覆盖区域丨中国29个省市（港澳台、西藏、海南除外）

核心问题丨系统地监测社区社会结构和家庭、劳动力个体的变化与相互影响

应用主题丨人口健康分析、劳动就业分析、消费储蓄分析、空间规划分析

# 4 中国家庭收入调查（CHIP）
中国家庭收入调查（CHIP）

网址：http://www.ciidbnu.org/chip/index.asp

为了追踪中国收入分配的动态情况，中国家庭收入调查（CHIP）已经相继在1989年、1996年、2003年、2008年和2014年进行了五次入户调查。它们分别收集了1988、1995、2002、2007和2013年的收支信息，以及其他家庭和个人信息，分别编号为CHIP1988、CHIP1995、CHIP2002、CHIP2007和CHIP2013。这几次调查是由中外研究者共同组织的、关于“中国收入和不平等研究”的组成部分，并且在国家统计局的协助下完成。CHIP项目的参与者和其他学者分析了这四次调查数据，并且发表了涉及很多领域的文章、报告和学术书籍。

所有的CHIP数据均包含针对城镇和农村住户的调查。鉴于农村向城镇迁移的日渐重要的现实意义，以及城镇和农村住户的子样本并不完全覆盖所有流动人口，2002年的调查增加了对流动人口的调查。因此，2002年CHIP调查包含了三个子样本。2007年的调查也采用了同样的方法，因此也由三个部分组成：城镇住户调查、农村住户调查和流动人口调查。这一结构反映了中国的城乡分割和近20年中不断增加的迁移到城镇地区的农村个体数量。

# 5 中国家庭追踪调查数据CFPS
中国家庭追踪调查数据CFPS

执行机构丨北京大学中国社会科学调查中心数据

网址丨http://www.isss.edu.cn/cfps/

“中国家庭追踪调查“（CFPS）重点关注中国居民的经济与非经济福利，以及包括经济活动、教育成果、家庭关系与家庭动态、人口迁移、健康等在内的诸多研究主题，是一项全国性、大规模、多学科的社会跟踪调查项目。CFPS样本覆盖25个省/市/自治区，目标样本规模为16000户，调查对象包含样本家户中的全部家庭成员。CFPS在2008、2009两年在北京、上海、广东三地分别开展了初访与追访的测试调查，并于2010年正式开展访问。经2010年基线调查界定出来的所有基线家庭成员及其今后的血缘/领养子女将作为CFPS的基因成员，成为永久追踪对象。

开放数据年份丨2008、2009（测试性调查，北京、上海、广东）；2010（基线调查）；2011（维护调查）；2012年以后每年一次跟踪调查。最新公开数据：CFPS2016（追访）调查数据。

数据类型丨面板数据

分析单位与调查规模丨社区、家庭、个人（成人、少儿）；基线调查为16000户。CFPS调查问卷共有社区问卷、家庭问卷、成人问卷和少儿问卷四种主体问卷类型，并在此基础上不断发展出针对不同性质家庭成员的长问卷、短问卷、代答问卷、电访问卷等多种问卷类型。

覆盖区域丨中国25个省市，2010年在全国（西藏、青海、新疆、宁夏、内蒙古、海南、香港、澳门、台湾不在其列）正式实施。

核心问题丨中国社会、经济、人口、教育和健康的变迁

应用主题丨人口健康分析、劳动就业分析、消费储蓄分析、空间规划分析、质量管理

主要调查项目：

家庭：生活条件、家户各类收入与支出、住房、金融资产等

成人：基本信息、教育、婚姻、工作、健康、退休与养老、认知、宗教等

少儿：基本信息、日常生活、健康、教育、培训辅导、认知能力等

其中，村/居问卷的调查内容包括：村/居基础设施概况、人口和劳动力资源概况、自身及周边环境、基层选举、财政收入与支出，以及日常消费品价格等。

家庭问卷的调查内容包括：家庭成员结构、日常生活基本设施、社会交往、住房、家庭经济、农业生产与销售等。

成人问卷的调查内容包括：教育、婚姻、职业、日常生活、健康、养老、社会保障、社会交往、价值观、以及基准测试等。

少儿问卷的调查内容包括：学业情况、日常生活、健康、职业期望、与父母关系、成长环境、社会交往、价值观、以及基准测试等。

# 6 中国健康与养老追踪调查(CHARLS)
中国健康与养老追踪调查(CHARLS)

执行机构丨北京大学中国社会科学调查中心数据

网址丨http://charls.pku.edu.cn/zh-CN

开放数据年份丨2008、2012（两省），2011、2013、2014（全国）

2011年（基线调查）；以后每两年追踪一次，调查结束1年后，数据对外界公开。2013年（追踪调查）；2014年（“中国中老年生命历程调查”专项）。最新公开数据：2015年CHARLS全国追踪调查数据。

数据类型丨面板数据

分析单位丨个人、家庭

覆盖区域丨基线调查在全国28个省的150个县区的450个村、居展开。浙江、甘肃两省（2008、2012），中国28个省市(2011、2013、2014)

核心问题丨我国人口老龄化问题

应用主题丨人口健康分析、消费储蓄分析

分析单位与调查规模：家户、个人（45岁及以上）；2015年全国追访时，其样本已覆盖总计1.24万户家庭中的2.3万名受访者。

主要调查项目：个人基本信息，家庭结构和经济支持，健康状况，体格测量，医疗服务利用和医疗保险，工作、退休和养老金、收入、消费、资产，以及社区基本情况等。

研究主题：人口老龄化问题、劳动经济学（婚姻、彩礼等）、社会保障、人口经济学、卫生经济学等。

# 7 中国家庭金融调查数据CHFS
中国家庭金融调查数据CHFS

执行机构丨西南财经大学

数据网址丨http://chfs.swufe.edu.cn/

开放数据年份丨2011年开始首轮调查，每两年进行一次追踪调查。目前可利用数据CHFS2011、CHFS2013、CHFS2015。

数据类型丨截面数据

分析单位丨家庭

覆盖区域丨25个省市（2011），29个省市（2013）

以CFPS2013为例，除追访2011年访问的8438户家庭、29000个个体外，样本进行首次扩展，最终共计调查来自全国29个省市、自治区（新疆、西藏除外）262个县区的28241个家庭，93000个个体。

核心问题丨家庭金融状况、收入支出、社会保障、商业保险等

应用主题丨人口健康分析、劳动就业分析、消费储蓄分析、金融与投资分析

# 8 全国老年人口健康状况调查研究（CLHLS）

全国老年人口健康状况调查研究（CLHLS），是由北京大学健康老龄与发展研究中心/国家发展研究院组织的老年人追踪调查，调查范围覆盖全国23个省区市，调查对象为65岁及以上老年人和35-64岁成年子女，调查问卷分为存活被访者问卷和死亡老人家属问卷两种。存活被访者问卷的调查内容包括老人及家庭基本状况、社会经济背景及家庭结构、经济来源和经济状况、健康和生活质量自评、认知功能、性格心理特征、日常活动能力、生活方 式、生活照料、疾病治疗和医疗费承担；死亡老人家属问卷的调查内容包括死亡时间、死因等内容。

# 9 中国健康与营养调查（CHNS）

中国健康与营养调查（China Health and Nutrition Survey, CHNS）是由北卡罗来纳大学人口研究中心（The Carolina Population Center at the University of North Carolina at Chapel Hill）﹑美国国家营养与食物安全研究所（The National Institute of Nutrition and Food Safety）和中国疾病与预防控制中心（The Chinese Center for Disease Control and Prevention）合作开展的调查项目。该调查旨在检验健康﹑营养和计划生育政策的影响以及研究中国社会经济的转变如何作用于整个人口健康和营养状况。到目前为止，该调查一共进行了7次，分别是1989﹑1991﹑1993﹑1997﹑2000﹑2004和2006年。该调查采用多阶段整群抽样的方法，其中有几年因为一些原因，调查的省份发生了变化，最新的2006年的调查范围涉及辽宁﹑黑龙江﹑江苏﹑山东﹑河南﹑湖北﹑湖南﹑广西和贵州9个省（自治区），调查内容涉及住户、营养、健康、成人、儿童、社区等。

**Comment by hzp2333** [comment]
CMDS已经挂掉了
